app/default_overrides.go

```diff
@@ -156,6 +199,15 @@ func DefaultBlockParams() tmproto.BlockParams {
 	}
 }
 
+// DefaultEvidenceParams returns a default EvidenceParams with a MaxAge
+// determined using a goal block time.
+func DefaultEvidenceParams() tmproto.EvidenceParams {
+	evdParams := coretypes.DefaultEvidenceParams()
+	evdParams.MaxAgeDuration = appconsts.DefaultUnbondingTime
+	evdParams.MaxAgeNumBlocks = int64(appconsts.DefaultUnbondingTime.Seconds())/int64(appconsts.GoalBlockTime.Seconds()) + 1
```

Comment:
[question] does this need to be updated?

https://github.com/celestiaorg/celestia-app/blob/4e62d2d3660e8ad5b6115348bed5aa6a57cf93fc/specs/src/specs/params.md?plain=1#L26

Reply:
updated! thanks